src/CodeGen_LLVM.cpp

```diff
@@ -3172,7 +3172,7 @@ void CodeGen_LLVM::visit(const Call *op) {
         }
 
         // Create a null-initialized global to track this object.
-        const auto base_fn = sub_fns.back().fn;
```

Comment:
I suspect the original was a typo and the intent was for it to be `const auto *`